src/Backend/Component/Core/Dashboard.php

```diff
@@ -153,8 +154,21 @@ public function checkProdMode(): string
             if (empty($rootPage->language)) {
                 $this->addError(sprintf($GLOBALS['TL_LANG']['WEMSG']['CORE']['DASHBOARD']['checkProdModeRootpageLanguageMissing'], $rootPage->title));
             }
-            if (empty($rootPage->sitemapName)) {
-                $this->addError(sprintf($GLOBALS['TL_LANG']['WEMSG']['CORE']['DASHBOARD']['checkProdModeRootpageSitemapMissing'], $rootPage->title));
+            // only for Contao < 4.11
```

Comment:
Why should we check for Contao versions < 11 if our composer requires a Contao 4.13?

Reply:
Because I thought we were still requiring Contao >= 4.9 🤦

Reply:
Fixed in https://github.com/Web-Ex-Machina/contao-smartgear/pull/279/commits/38830bd517363647f7c6d8bb9c656855073ecc1b